gulpfile.js

```diff
@@ -1,26 +1,24 @@
-'use strict';
```

Comment:
Sure, I'll leave it then. The easy fix to all this would just be to use a `gulpfile.ts` instead too, but I don't really want to worry about getting the definitions working. Why do you use gulp when the same thing could be a couple of CLI scripts in `package.json`?